tools/topology/m4/muxdemux.m4

```diff
@@ -43,9 +43,7 @@ define(`W_MUXDEMUX',
 `SectionVendorTuples."'N_MUXDEMUX($1)`_tuples_uuid" {'
 `	tokens "sof_comp_tokens"'
 `	tuples."uuid" {'
-ifelse(`$2', `0',
-`		SOF_TKN_COMP_UUID'	STR(mux_uuid),
-`		SOF_TKN_COMP_UUID'	STR(demux_uuid))
+`ifelse(`$2', `0',`		SOF_TKN_COMP_UUID'	STR(mux_uuid),`		SOF_TKN_COMP_UUID'	STR(demux_uuid))'
```

Comment:
so it's actually a pair of ` and' to include the whole ifelse sentence is needed, no matter how much lines it occupies?

Reply:
@keyon I guess so, I noticed that the string "mux/demux" setting below works, but this part didn't. I was debugging this for 3 days, since the string shows it is "MUX", but I never looked at the uuid. In the fw mux/demux share almost all function pointers, only copy function is different -> when copy crashed, the log was missing the last bits -> I didn't understand I was actually running demux instead of mux...